stories/info/atoms/buttonStrip.md

```diff
@@ -0,0 +1,20 @@
+# ButtonStrip
+
+A ButtonStrip is a framing element that can be used to layout an inline set of `Buttons`. This component will only accept instances of the `Button` component as its children and requires at least 2 children.
+
+## Usage
+
+In its default configuration, the `ButtonStrip` will add 16px of whitespace between the buttons, and it will align them on the left (flex-start). It also accepts the following properties:
+
+| Prop       | Type      | Description                                                    |
```

Comment:
The prop table is generated automatically and shown in a tab in storybook. No need to have a manually updated version.